varinstall/yum-repos/centreon-20.04-el7.repo

```diff
@@ -0,0 +1,63 @@
+[centreon-stable-noarch]
```

Comment:
https://github.com/centreon/centreon/pull/9761/files#diff-2b45a64fb59d94a79d816de404df8c4f30bfccff9cd0b2a94fe4e6d623e2a7f9R7

Are 20.04 versions also supported ?

Reply:
It will be possible to use those files instead of the centreon-release rpm for the 20.04 ; even if this version of the unattended.sh script will not manage this version.